app/src/main/java/org/simple/clinic/widgets/PatientSearchResultItemView.kt

```diff
@@ -93,6 +107,48 @@ class PatientSearchResultItemView(
     renderPatientPhoneNumber(searchQuery, model)
     renderVisited(model.lastSeen)
     renderLastSeen(model.lastSeen, currentFacilityId)
+    renderIdentifier(model.identifier, searchQuery)
+  }
+
+  private fun getId(identifier: Identifier, searchQuery: String): Id {
+    val id = when (identifier.type) {
+      BpPassport -> BpPassport.shortCode(identifier)
+      BangladeshNationalId -> identifier.value
+      EthiopiaMedicalRecordNumber -> identifier.value
+      IndiaNationalHealthId -> identifier.value
+      is Unknown -> throw IllegalArgumentException("Unknown identifier: $identifier")
+    }
+    val canHighlightIdentifier = id.contains(searchQuery)
+
+    return if (canHighlightIdentifier) {
```

Comment:
Can't we skip the method call for `contains` and use the `indexOf` return value to check if the search query is present in the search result?